alpaca_trade_api/polygon/entity.py

```diff
@@ -343,7 +343,10 @@ class DailyOpenClose(Entity):
     "x": "exchange",
     "p": "price",
     "s": "size",
-    "t": "timestamp"
+    "t": "timestamp",
+    "f": "trf_timestamp",
+    "y": "exchange_timestamp",
```

Comment:
we use `participant_timestamp` not `exchange_timestamp` so should be probably changed to that

Reply:
there are some other fields we should consider map: i, r, q, z

Reply:
I renamend `exchange_timestamp` to `participant_timestamp` and added `i`, `r`, `q` and `z` to mapping.